### PR TITLE
Channel Area Business Logic

### PR DESCRIPTION
**Pull request by MattReimer**
This pull request is for completing the channel area business logic.

Currently this change is sitting on the `ChannelArea` branch of this repo so all you need to do is check it out, make the changes then push back.

### CODE REVIEW

**Review by joewheaton — APPROVED** [review]
Looks like a great start! I like.

**Review by MattReimer — COMMENTED** [review]
Looks good to me @lauren-herbine is this ready to be merged?

**Comment by lauren-herbine** [comment]
@MattReimer I got a bit confused about the already created pull request and don't think I did the whole commit/pull thing correctly... but I had @shelbysawyer review this in person and she is good with it. I was just going to add @joewheaton as a reviewer (though he saw a bit in person as well) and add some screenshots of the symbology to satisfy our template checklist for curation (#166)

TL;DR- yes, this is ready to be merged, but let's wait for the official blessing from @joewheaton

**Comment by MattReimer** [comment]
Cool, yeah. All good @lauren-herbine. I got confused that @joewheaton had reviewed this but it seemed like there was still work to do. No harm in it and checking in to be sure is fine. I've clicked the button for Joe to re-review your latest changes. Feel free to add screenshots or whatever you need to 

@joewheaton I'll merge this the next time you approve it.

**Comment by lauren-herbine** [comment]
Great, thanks @MattReimer! I'll add some documentation below @joewheaton

**Review by joewheaton — APPROVED** [review]

**Comment by lauren-herbine** [comment]
Here is a video showing the symbology and business logic- felt more efficient than 15+ screenshots: 

[Alt text](https://www.youtube.com/watch?v=VID)

And here are the layers:
image

And this is what the basic project view looks like (hollow and filled polygons turned on):
image

**Comment by joewheaton** [comment]
It's funny, when you were showing @shelbysawyer today, I sort of thought to myself for big ones a video might be nicer some times.

**Comment by shelbysawyer** [comment]
@lauren-herbine I don't want to mess anything up by playing with it now, but once this gets merged, can we move the symbology files for `ChannelArea `into shared so VBET and TauDEM can both pull the symbology for it?

**Comment by lauren-herbine** [comment]
@shelbysawyer for sure

**Comment by lauren-herbine** [comment]
Hi @MattReimer , I am having some issues getting this symbology into the shared folder for qgis. I created a new branch and am trying to copy and paste in my file manager so that I can create a new pull request- is this not the best way to go about it?

**Comment by MattReimer** [comment]
Hi @lauren-herbine I would move instead of copy. If something is in the SHared folder we really don't need it anywhere else.... Maybe I'm reading the question wrong. 

Also make sure. your new branch is starting of where master currently is so you can see all the files you need (I assume that's what you did but never hurts to be more specific)

**Comment by lauren-herbine** [comment]
@MattReimer Ok, I recorded a quick vid, as I don't think I'm explaining well and I feel like this is something simple I'm just missing:

[Alt text](https://youtu.be/5Z3vX5D2yuI)

**Comment by MattReimer** [comment]
Cool. I'm away from my desk until this afternoon but I'll take a look as soon as I get back.

**Comment by lauren-herbine** [comment]
Awesome, thanks, no rush!!

**Comment by MattReimer** [comment]
Sorry @lauren-herbine this kind of fell off my radar. It looks like the deletion was registered but not the move so all I had to do was roll git back to before the deletion and copy all the files over to the Shared branch then re-commit them. 

Thank goodness for git. It's annoying at times but it really takes the stress out of accidentally deleting things. 

Anyway give this a try. All the QML should be in the right place now.

**Comment by lauren-herbine** [comment]
Thank you so much for figuring this out! I just checked and it looks like everything is in the right place now. Closing this
